README.md

```diff
@@ -23,8 +21,9 @@ Additional description about the project and its features.
 - ![](https://img.shields.io/badge/HTML-purple) 
 - ![](https://img.shields.io/badge/CSS-blue)
 - ![](https://img.shields.io/badge/WEBPACK-violet)
-- barbel
-- jest
+- ![](https://img.shields.io/badge/Barbel-violet)
+- ![](https://img.shields.io/badge/API-violet)
+- ![](https://img.shields.io/badge/JSON-violet)
 
 ##periquities
```

Comment:
[OPTIONAL] There is no need to invalidate your PR for this,
Your README file looks professional
Kindly fix this typo ^^!

and make sure in your future submits that your README is well made like the one you having here 👍🏽